src/GDGUkraine/errors.py

```diff
@@ -172,10 +172,10 @@ def get_error_page(status, errors=None, **kwargs):
                 if cherrypy.lib.is_iterator(result):
                     from cherrypy.lib.encoding import UTF8StreamEncoder
                     return UTF8StreamEncoder(result)
-                elif isinstance(result, cherrypy._cpcompat.unicodestr):
+                elif isinstance(result, str):  # str is OK for Python3
                     return result.encode('utf-8')
```

Comment:
I'd not encode here as cherrypy does all the stuff internally, relying on default encoding set: https://github.com/cherrypy/cherrypy/blob/v7.1.0/cherrypy/lib/encoding.py#L113

Instead I suggest doing the following:
```python
from cherrypy.lib.encoding import bytestring
...
elif isinstance(result, bytestring):  # this catches both bytes and str
    return result  # cherrypy will encode chunk by chunk if needed
else:
    raise that error
```